### ISSUE TITLE
network inspect output is non-intuitive

### ISSUE THREAD

**Issue by cpuguy83** [docker/swarm#1373]
Since a multi-host network is by definition available on every node of the cluster, there is only one entry in `docker network ls`.
However, if I `docker network inspect multi` I only get output from a single engine:

```json
[
    {
        "Name": "multi",
        "Id": "5580acd70dd89d58cecd16df769ace923c91226ce9d6e22828ec83efd8a25c46",
        "Scope": "global",
        "Driver": "overlay",
        "IPAM": {
            "Driver": "default",
            "Config": [
                {}
            ]
        },
        "Containers": {
            "eaf4bc7e2f99fd3b82e7647ec449cd515cc1d53dffe3a037fa877121ce6f6508": {
                "EndpointID": "4de7a3ba418425e2e1a9ba7191ae8e302c56132163ddb7fa6ac65334a3e00e53",
                "MacAddress": "02:42:0a:00:01:02",
                "IPv4Address": "10.0.1.2/24",
                "IPv6Address": ""
            }
        },
        "Options": {}
    }
]
```

In reality I have 3 containers attached to this network, however there doesn't seem to be anyway to actually get at that information.

**Comment by abronan** [comment]
Yes we discussed about that in #1262 but this couldn't make it in time for `1.0.0`. This will likely be included in a `1.0.1` with some small enhancements.

**Comment by abronan** [comment]
Closed by #1547
